.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+files: simulariumio
```

Comment:
If you wanted to add `black` `flake8` and `mypy`, this is where they would go. Let me know when you want to do that.